docker/data-source-services/docker-compose.yml

```diff
@@ -39,6 +39,22 @@ services:
       - MYSQL_PASSWORD=mariadb_pass
     networks:
       - st-internal
+  minio:
```

Comment:
Can we make this a `data-source-tools` service instead of a `data-source-services` service?

That way it is only started by people who want the additional tools and not just the required services needed for development.